src/pages/es/index.astro

```diff
@@ -0,0 +1,172 @@
+---
+import config_json from "../../review_templates/template_es.json";
+import PageLayout from "../../layouts/PageLayout.astro";
+---
+
+<PageLayout title="Plantilla de reseñas de steam">
+  <header class="bg-slate-700 text-white py-2 flex flex-col items-center">
+    <h1>{"---{ Plantilla para reseñas }---"}</h1>
+    <a href="/steam-review-template/en/" class="mt-0 text-white">Sitio en ingles</a>
```

Comment:
Instead of this subtitle line, can we maybe do emojis in the corner? Something more subtle 🇪🇸 🇬🇧